ckanext/xloader/templates/package/snippets/resource_item.html

```diff
@@ -0,0 +1,8 @@
+{% ckan_extends %}
+
+{% block resource_item_explore_inner %}
```

Comment:
This block exists only in CKAN core master, not in any released version, and so it will have no effect.

Reply:
Added max 2.10 version check in other block